.kitchen.yml

```diff
@@ -31,7 +31,7 @@ platforms:
       box: windows-8.1
       box_url: http://cdn.box-cutter.com/windows/virtualbox4.3.20/eval-win81x64-enterprise-nocm-1.0.2.box
     provisioner:
-      require_chef_omnibus: 12.0.0
+      require_chef_omnibus: 12
```

Comment:
No. The nix stuff behaves the same way iirc

On Wed, Jan 28, 2015, 9:38 PM Matt Stratton <notifications@github.com>
wrote:

> In .kitchen.yml
> <https://github.com/opscode-cookbooks/windows/pull/172#discussion_r23744912>
> :
>
> > @@ -31,7 +31,7 @@ platforms:
> >        box: windows-8.1
> >        box_url: http://cdn.box-cutter.com/windows/virtualbox4.3.20/eval-win81x64-enterprise-nocm-1.0.2.box
> >      provisioner:
> > -      require_chef_omnibus: 12.0.0
> > +      require_chef_omnibus: 12
>
> Do we want to wait for this to be fixed in test-kitchen before merging
> this? Seems logical.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opscode-cookbooks/windows/pull/172/files#r23744912>.
>